### PR TITLE
Touch Bar API sample

### PR DESCRIPTION
**Pull request by Rawnly**
Added electron-touchbar-example 

> Sample  touchbar integration

**By submitting this pull request, I promise I have read the [contributing guidelines](https://github.com/sindresorhus/awesome-electron/blob/master/contributing.md) twice and ensured my submission follows it. I realize not doing so wastes the maintainers' time that they could have spent making the world better. 🖖**

### CODE REVIEW

**Comment by sindresorhus** [comment]
It's a good idea to show off how to use the Touch Bar APIs, but your project just has way too much needless boilerplate. I'd like to see a simple setup like https://github.com/hokein/electron-sample-apps/tree/master/cookies Samples should just be possible to run right away without any boilerplate or prebuild-step.

File: readme.md [hunk 1]
```diff
@@ -217,6 +217,7 @@ Made with Electron.
 - [Activity Monitor](http://electron.atom.io/blog/2017/01/19/simple-samples#activity-monitor) - Shows a doughnut chart of the CPU system, user, and idle activity time.
 - [Hash](http://electron.atom.io/blog/2017/01/19/simple-samples#hash) - Shows the hash values of entered text using different algorithms.
 - [Prices](http://electron.atom.io/blog/2017/01/19/simple-samples#prices) - Shows the current price of oil, gold, and silver using the Yahoo Finance API.
+- [TouchBar API Sample](https://github.com/Rawnly/electron-touchbar-example) - Sample touchbar integration.
```
Comment (sindresorhus) [inline readme.md]: It's `Touch Bar`, not `TouchBar` or `touchbar`. The word `Sample` is also moot as it's in the Sample section.

Comment (Rawnly) [reply readme.md]: 👍

**Comment by Rawnly** [comment]
Oh ok! I'll keep the file structure simple and easy to understand without any prebuild process

**Comment by Rawnly** [comment]
@sindresorhus I've updated it! Check it out now it's really simple, only 3 files and all the code is commented. 

I've added all the `TouchBar Objects` in the documentation, from `label` to `scrubber`.

Now render process is only *1 html file*, with a couple of `CSS` and `JS` (to catch touch bar events via `IPC`)

**Comment by sindresorhus** [comment]
You still need to update https://github.com/sindresorhus/awesome-electron/pull/490#discussion_r147332633 and your Touch Bar name usage in your own readme.

**Comment by Rawnly** [comment]
👌@sindresorhus

**Comment by sindresorhus** [comment]
Still not correct. Please read my comment more thoroughly.

**Comment by Rawnly** [comment]
What do you mean? I’ve fixed _TouchBar_ to _Touch Bar_ in the **fork** and in the project README.. where else should I fix?

Federico Vitale

Il 6 nov 2017, 06:18 +0100, Sindre Sorhus <notifications@github.com>, ha scritto:

**Comment by Rawnly** [comment]
What do you mean? I’ve fixed _TouchBar_ to _Touch Bar_ in the **fork** and in the project README.. where else should I fix?

**Comment by sindresorhus** [comment]
Both times Touch Bar is written in your entry, it's written incorrectly and differently:

```
> [Touch bar API Sample](Rawnly/touchbar-api-sample) - Sample touchbar integration.
```

**Comment by sindresorhus** [comment]
You still need to update your readme description: `Electron sample touch bar API` => `Example of using the Touch Bar with Electron`

**Pull request merged**